Resources/Prototypes/Catalog/Fills/Lockers/medical.yml

```
@@ -70,6 +71,41 @@
         prob: 0.3
       - id: ClothingMaskSterile
 
+- type: entity
+  id: LockerWallMedicalDoctorFilled
+  suffix: Filled
+  parent: LockerWallMedical
+  components:
+  - type: StorageFill
+    contents:
+      - id: HandheldHealthAnalyzer
+        prob: 0.6
+      - id: ClothingHandsGlovesLatex
+      - id: ClothingHeadsetMedical
+     # - id: ClothingEyesHudMedical #Removed until working properly
+      - id: ClothingBeltMedical
+      - id: ClothingHeadHatSurgcapGreen
+        prob: 0.1
+        orGroup: Surgcaps
+      - id: ClothingHeadHatSurgcapPurple
+        prob: 0.05
+        orGroup: Surgcaps
+      - id: ClothingHeadHatSurgcapBlue
+        prob: 0.90
+        orGroup: Surgcaps
+      - id: UniformScrubsColorBlue
+        prob: 0.5
+        orGroup: Surgshrubs
+      - id: UniformScrubsColorGreen
+        prob: 0.1
+        orGroup: Surgshrubs
+      - id: UniformScrubsColorPurple
+        prob: 0.05
+        orGroup: Surgshrubs
+      - id: NitrousOxideTankFilled
+        prob: 0.3
+      - id: ClothingMaskSterile
```

Dont copy paste like this, im pretty sure you can use prototype composition here if you want to inherit both `LockerWallMedical` sprites and `LockerMedicalFilled` fill, also name it properly
```suggestion
- type: entity
  id: LockerWallMedicalFilled
  parent: ["LockerWallMedical", "LockerMedicalFilled"]
```

---

I can't change the ID names as they are most likely already mapped on several maps

---

I actually don't know what you want me to do here? I keep looking at it and it's how it's done everywhere else.

---

Oh, sorry, I didn't see that bit at the bottom. of your comment.